templates/blockonomics_checkout.php

```diff
@@ -120,6 +122,12 @@
         </div>
       </div>
     </div>
+    <!-- BCH address Generation Error -->
+    <div class="bnomics-display-error" style="text-align: center;" ng-show="bchaddresserror">
+            <h4>Address Generation Error</h4>
+            <h4><i class="material-icons bnomics-alt-icon">error</i></h4>
+            <p>Note to webmaster: Please follow the instructions <a href="https://help.blockonomics.co/support/solutions/articles/33000251576-bch-setup-on-whmcs" target="_blank">here</a> to configure BCH payments.</p>
```

Comment:
Incorrect href. Please change this to https://help.blockonomics.co/support/solutions/articles/33000253348-bch-setup-on-woocommerce. This freshdesk article will be updated accordingly.